src/components/Kv/KvSocialShareButton.vue

```diff
@@ -21,55 +21,46 @@
 		>
 			<slot name="modal-content"></slot>
 			<div
-				class="tw-inline-grid tw-grid-cols-1 md:tw-grid-cols-2 tw-gap-1 tw-mt-2.5"
+				class="tw-flex tw-mt-2.5 tw-flex-wrap"
 			>
 				<kv-button
 					variant="ghost"
 					class="social-button"
 					data-testid="share-facebook-button"
-					:href="facebookShareUrl"
-					target="_blank"
-					rel="noopener"
 					v-kv-track-event="[trackingCategory, 'share', 'facebook', utmCampaign, loanId]"
-					@click="$showTipMsg('Thanks for sharing to Facebook!')"
+					@click="showSharePopUp(facebookShareUrl, 'Thanks for sharing to Facebook!')"
 				>
 					<kv-material-icon
 						class="social-button__icon social-button__icon--facebook"
 						:icon="mdiFacebook"
 					/>
-					<span class="tw-font-medium">Share on Facebook</span>
```

Comment:
Removed "Share on" and modified css to allow or the 4 buttons to fit on one line, see screenshot